### DISCUSSION TITLE
Data dump output via mail

### DISCUSSION THREAD

**Discussion by moritzvolkmar** [Samedis-care/samedis-care-community#53] [Shipped - Live on production]
Creating a data dump takes quite long depending on the size of the tenant. Currently, you only get the message that it is ready via notifications (bell) at some point. 

Idea:

1. start data export as usual
2. when finished, mail with download link to registered user
3. adjust infoboxes/dialogs accordingly

**Comment by neongrau** [comment]
Mail would be easy to implement.

I'm just not sure the scenario why the dump was requested is meant for this. We just automated a DSGVO required dump that we could as well manually create and send to the requesting party.

We probably just need some other exports that would be faster and easier to get.

**Comment by YvesR** [comment]
Mail sent for individual messages coming with next upd.
